### DISCUSSION TITLE
List of GitHub repositories with "publiccode.yml"

### DISCUSSION THREAD

**Discussion by fnogatz** [publiccodeyml/publiccode.yml#151] [Show and tell]
Hey folks, I'm not sure if you have been aware of this (and if this is the right place to point you there), but it might be of interest for you: @tomsrocket created a Python script to retrieve and parse all "publiccode.yaml" files from github repositories. The source code is available at https://github.com/tomsrocket/github-publiccode (without `publiccode.yml`, sorry :laughing: ). It also features a table that has a list of (nearly) all repositories with publiccode.yml files. See the list of all projects here: https://tomsrocket.github.io/github-publiccode/public/

**Comment by bzg** [comment]
Nice! Cc'ing @garronej on this one, as he might be interested too.

FWIW, we plan to list repos with `publiccode.yml` files on code.gouv.fr as well, but we're not sure when, our plate is quite full at the moment.
